### PR TITLE
Feature/ch5616

### PR DESCRIPTION
**Pull request by ryuheimat**
[ch5616](https://app.clubhouse.io/masterventures/story/5616/as-a-front-end-developer-i-need-to-modify-the-dapp-main-menu-according-to-the-figma-design)

### CODE REVIEW

File: redux/actions/template/saft.html.ts [hunk 1]
```diff
@@ -49,13 +49,20 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
...
 <p class="western" align="justify">&nbsp;</p>
 <p class="western" align="justify">(e) In the event any one or more of the provisions of this instrument is for any reason held to be invalid, illegal or unenforceable, in whole or in part or in any respect, or in the event that any one or more of the provisions of this instrument operate or would prospectively operate to invalidate this instrument, then and in any such event, such provision(s) only will be deemed null and void and will not affect any other provision of this instrument and the remaining provisions of this instrument will remain operative and in full force and effect and will not be affected, prejudiced, or disturbed thereby.</p>
 <p class="western" align="justify">&nbsp;</p>
+<<<<<<< HEAD
```
Comment (92Sam) [inline redux/actions/template/saft.html.ts:61]: conflict here my friend

File: redux/actions/template/saft.html.ts [hunk 2]
```diff
@@ -80,11 +87,19 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
 <p class="western">&nbsp;</p>
 <p class="western">&nbsp;</p>
 </div>
+<<<<<<< HEAD
```
Comment (92Sam) [inline redux/actions/template/saft.html.ts:90]: conflict here

**Review by 92Sam — APPROVED** [review]
LGTM! Please recieve the messages

**Review by CoolHandLuke88 — CHANGES_REQUESTED** [review]
@ryuheimat 
~~this pr looks like it was not rebased from main, unless I am missing something, 193 commits, I believe you need to rebase with your changes and then open a new PR.~~

See comments regarding merge conflict.

Thank you sir

File: redux/actions/template/saft.html.ts [hunk 3]
```diff
@@ -49,16 +49,20 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
...
-<p class="western" align="justify">6. <em><strong>Miscellaneous</strong></em></p>
-<p class="western" align="justify">(a) This instrument sets forth the entire agreement and understanding of the parties relating to the subject matter herein and supersedes all prior or contemporaneous disclosures, discussions, understandings and agreements, whether oral of written, between them. This instrument is one of a series of similar instruments entered into by the Company from time to time. Any provision of this instrument may be amended, waived or modified only upon the written consent of the Company and the holders of a majority, in the aggregate, of the Purchase Amounts paid to the Company with respect to all SAFTs outstanding at the time of such amendment, waiver or modification.</p>
+=======
```
Comment (CoolHandLuke88) [inline redux/actions/template/saft.html.ts:54]: sam as Sams previous comment

File: redux/actions/template/saft.html.ts [hunk 4]
```diff
@@ -49,16 +49,20 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
...
 <p class="western" align="justify">&nbsp;</p>
 <p class="western" align="justify">(e) In the event any one or more of the provisions of this instrument is for any reason held to be invalid, illegal or unenforceable, in whole or in part or in any respect, or in the event that any one or more of the provisions of this instrument operate or would prospectively operate to invalidate this instrument, then and in any such event, such provision(s) only will be deemed null and void and will not affect any other provision of this instrument and the remaining provisions of this instrument will remain operative and in full force and effect and will not be affected, prejudiced, or disturbed thereby.</p>
 <p class="western" align="justify">&nbsp;</p>
+<<<<<<< HEAD
 <p class="western" align="justify">(f) All rights and obligations hereunder will be governed by the laws of <div class="underline-contract-line">{{#jurisdiction}}{{jurisdiction}}{{/jurisdiction}}{{^jurisdiction}}[JURISDICTION]{{/jurisdiction}}</div>, without regard to the conflicts of law provisions of such jurisdiction.</p>
+=======
```
Comment (CoolHandLuke88) [inline redux/actions/template/saft.html.ts:63]: same as @92Sam previous comment, merge conflict.

File: redux/actions/template/saft.html.ts [hunk 5]
```diff
@@ -83,11 +87,19 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
 <p class="western">&nbsp;</p>
 <p class="western">&nbsp;</p>
 </div>
+<<<<<<< HEAD
 <p class="western" align="center">&nbsp;</p>
 <p class="western" align="center"><u><strong>EXHIBIT A</strong></u></p>
 <p class="western" align="center">&nbsp;</p>
 <p class="western" align="center"><span style="font-size: large;"><strong>Payment Instructions</strong></span></p>
 <p class="western" align="center">&nbsp;</p>
+=======
```
Comment (CoolHandLuke88) [inline redux/actions/template/saft.html.ts:96]: same as @92Sam previous comment, merge conflict.

File: redux/actions/template/saft.html.ts [hunk 6]
```diff
@@ -83,11 +87,19 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
...
+<p class="western" class="title-center">&nbsp;</p>
+<p class="western" class="title-center"><u><strong>EXHIBIT A</strong></u></p>
+<p class="western" class="title-center">&nbsp;</p>
+<p class="western" class="title-center"><span style="font-size: large;"><strong>Payment Instructions</strong></span></p>
+<p class="western" class="title-center">&nbsp;</p>
+>>>>>>> main
```
Comment (CoolHandLuke88) [inline redux/actions/template/saft.html.ts:102]: same as @92Sam previous comment, merge conflict.

File: redux/actions/template/saft.html.ts [hunk 7]
```diff
@@ -49,13 +49,20 @@ const Saft = `<div class="contract" style="width: 100%; background-color:white;
 <p class="western" align="justify">(h) The current structure of this SAFT and all transactions and activities contemplated hereunder is not a scheme to evade the registration requirements of the Securities Act.</p>
 <p class="western" align="justify">5. <em><strong>Procedures for Purchase of Rights and</strong></em> <em><strong>Valuation of Purchase Amount</strong></em><em>.</em></p>
 <p class="western" align="justify">(a) The Company will accept payment for the Right purchased under this SAFT in U.S. Dollars, Bitcoin and Ethereum. Purchaser shall make the required payment to the Company in consideration for Purchaser&rsquo;s purchase of the Right pursuant to the SAFT through the procedures set forth on <em><u>Exhibit A</u></em> hereof.</p>
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

File: redux/actions/template/saft.html.ts (hunk 1 above)
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

File: redux/actions/template/saft.html.ts (hunk 2 above)
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

File: redux/actions/template/saft.html.ts (hunk 3 above)
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

File: redux/actions/template/saft.html.ts (hunk 4 above)
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

File: redux/actions/template/saft.html.ts (hunk 5 above)
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

File: redux/actions/template/saft.html.ts (hunk 6 above)
Comment (ryuheimat) [reply redux/actions/template/saft.html.ts]: fixed

**Review by CoolHandLuke88 — APPROVED** [review]

**Pull request merged**